yarn.lock

```diff
@@ -3250,24 +3249,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"copyfiles@npm:^2.4.1":
```

Comment:
copyfiles module, which was used for copying ALS .js files to `out/server` directory is no longer needed.